dist/Storable/Makefile.PL

```diff
@@ -16,9 +16,13 @@ WriteMakefile(
     AUTHOR              => 'Perl 5 Porters',
     LICENSE             => 'perl',
     DISTNAME            => "Storable",
-# We now ship this in t/
-#    PREREQ_PM           => { 'Test::More' => '0.41' },
-    PREREQ_PM           => { XSLoader => 0 },
+    PREREQ_PM           => { 'Test::More' => '0.41' },
+    PREREQ_PM           =>
+      {
+          XSLoader => 0,
+          'Test::More' => '0.41',
+          'ExtUtils::MakeMaker' => '6.31',
```

Comment:
If we want to require EUMM before executing Makefile.PL,
we should put it in prereqs/configure/requires section in META files.

Reply:
I've updated Makefile.PL to set configure, build, and test requirements.